services/minespace-web/src/components/pages/Project/ProjectSummaryPage.tsx

```diff
@@ -315,18 +231,22 @@ export const ProjectSummaryPage: FC<ProjectSummaryPageProps> = (props) => {
     if (e) {
       e.preventDefault();
     }
+
     const message = newActiveTab
       ? "Successfully updated the project description."
       : "Successfully submitted a project description to the Province of British Columbia.";
-    const values = { ...formValues, status_code: "SUB" };
-    const missingRequiredFields = verifyRequiredFields(handleTransformPayload(values));
-    if (missingRequiredFields) {
-      handleTabChange(getRequiredFieldFormTab(missingRequiredFields));
-      return;
+
+    let status_code = projectSummary.status_code;
+    if (!status_code || !isEditMode) {
+      status_code = "DFT";
+    } else if (!newActiveTab) {
+      status_code = "SUB";
```

Comment:
I like this method of determining the submission status. No new page? We must be on the last one! 😂